modules/living_spaces_group/src/Access/LivingSpacesGroupPermissionChecker.php

```diff
@@ -39,7 +39,8 @@ public function __construct(GroupPermissionCheckerInterface $original, LivingSpa
    */
   public function hasPermissionInGroup($permission, AccountInterface $account, GroupInterface $group) {
     if ($this->livingSpacesMananger->isLivingSpace($group->bundle())) {
-      if (in_array('office_manager', $account->getRoles())) {
```

Comment:
Those permissions are in modules dependant to group module.
We can introduce a variable or something like that to allow to set permissions ignored by office manager in space or something like that